.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,26 @@
+name: CI Deploy storybook
+
+on:
+  release:
+    types: [published, unpublished]
```

Comment:
why unpublished ?

Reply:
To have storybook sync with the current version

Reply:
But even if you unrelease, it will just regenerate storybook from the master , so what's the difference ? (actually it would even be a more recent version than the previously released one ?)

Reply:
You right! I removed the `unpublished` trigger